index.js

```diff
@@ -15,14 +15,8 @@ var exclusions = ['--help', '--completion_bash'];
 // This number must be incremented whenever the generated cache file changes.
 var CACHE_VERSION = 2;
```

Comment:
Let's bump this too.
```suggestion
var CACHE_VERSION = 3;
```